src/components/progress/themes/shared.scss

```diff
@@ -0,0 +1,22 @@
+%base {
+    align-items: center;
+    justify-content: center;
+    font-family: var(--ig-font-family), serif;
+}
+
+%label {
+    display: flex;
+    align-items: center;
+    color: inherit;
+    font-weight: 600;
+}
+
+@mixin stripe-colors($primary, $gray, $size, $deg: -45deg) {
+    background: $primary repeating-linear-gradient(
+        $deg,
+        $primary,
+        $primary $size,
+        $gray $size,
+        $gray ($size * 2)
+    );
+}
```

Comment:
Having these is not necessary IMO. It Makes you jump between files just to read a few lines. Also, if you make the background property a CSS variable, this mixin is not needed.